sample.config.yaml

```diff
@@ -0,0 +1,131 @@
+  #  yourserver.com: supersecretsharedsecret
+  # optionally override the display name of the bridge bot
+  #displayname: Protocol Bot
+  # optionally set the avatar of the bridge bot
+  #avatarUrl: mxc://yourserver.com/somefile
+
+logging:
+  # Log level of console output
+  # Allowed values starting with most verbose:
+  # silly, debug, verbose, info, warn, error
+  console: info
+  # Optionally, you can apply filters to the console logging
+  #console:
+  #  level: info
+  #  enabled:
+  #    - Store
+  #  disabled:
+  #    - PresenceHandler
+
+  # Date and time formatting
+  lineDateFormat: MMM-D HH:mm:ss.SSS
+  # Logging files
+  # Log files are rotated daily by default
+  files:
+    # Log file path
+    - file: "bridge.log"
+      # Log level for this file
+      # Allowed values starting with most verbose:
+      # silly, debug, verbose, info, warn, error
+      level: info
+      # Date and time formatting
+      datePattern: YYYY-MM-DD
+      # Maximum number of logs to keep.
+      # This can be a number of files or number of days.
+      # If using days, add 'd' as a suffix
+      maxFiles: 14d
+      # Maximum size of the file after which it will rotate. This can be a
+      # number of bytes, or units of kb, mb, and gb. If using the units, add
+      # 'k', 'm', or 'g' as the suffix
+      maxSize: 50m
+      # Optionally enable/disable logging for certain modules
+      #disabled:
+      #  - PresenceHandler
+      #  - module: bot-sdk-MatrixLiteClient
+      #    regex: /_matrix/client/r0/presence/ # this regex needs to match to disable the log
+      #enabled:
+      #  - Store
+
+database:
+  # Use Postgres as a database backend
+  # If set, will be used instead of SQLite3
+  # Connection string to connect to the Postgres instance
+  # with username "user", password "pass", host "localhost" and database name "dbname".
+  # Modify each value as necessary
+  #connString: "postgres://user:pass@localhost/dbname?sslmode=disable"
+  # Use SQLite3 as a database backend
+  # The name of the database file
+  filename: database.db
+
+provisioning:
+  # Regex of Matrix IDs allowed to use the puppet bridge
+  whitelist:
+    # Allow a specific user
+    #- "@user:server\\.com"
+    # Allow users on a specific homeserver
+    - "@.*:yourserver\\.com"
+    # Allow anyone
+    #- ".*"
+  # Regex of Matrix IDs forbidden from using the puppet bridge
+  #blacklist:
+    # Disallow a specific user
+    #- "@user:server\\.com"
+    # Disallow users on a specific homeserver
+    #- "@.*:yourserver\\.com"
+
+presence:
+  # Bridge online/offline status
+  enabled: true
+  # How often to send status to the homeserver in milliseconds
+  interval: 500
+  # if the im.vector.user_status state setting should be diabled
+  #disableStatusState: false
+  # A blacklist of remote user IDs for the im.vector.user_status state setting
+  #statusStateBlacklist:
+  # - baduser
+
+relay:
+  # Enable/Disable relay bridging mode
+  enabled: false
```

Comment:
uuuh, the `enabled` flag is an old key. Might be that soru forgot to remove it from the bridge library.

Reply:
Removed!